tests/common/plugins/pdu_controller/snmp_pdu_controllers.py

```diff
@@ -168,107 +168,107 @@ def map_host_to_lane(self, lane_id):
         upon the number of PSUs on DUT. The calculated mode is used as an index to get PDU ports ID stored in
-        self.pdu_ports. But still, we cannot gurante that psu_id 0 is first PSU of DUT, and so on.
+        self.pdu_ports. But still, we cannot gurante that outlet 0 is first PDU of DUT, and so on.
 
-        @param psu_id: ID of the PSU on SONiC DUT
+        @param outlet: ID of the PDU on SONiC DUT
         @return: Return true if successfully execute the command for turning off power. Otherwise return False.
         """
-        if not self.psuType:
-            logging.error('Unable to turn off: PSU type is unknown')
+        if not self.pduType:
+            logging.error('Unable to turn off: PDU type is unknown')
             return False
 
-        port_oid = self.pPORT_CONTROL_BASE_OID + self.pdu_ports[rfc1902.Integer(psu_id)]
+        port_oid = self.pPORT_CONTROL_BASE_OID + self.pdu_ports[rfc1902.Integer(outlet)]
         errorIndication, errorStatus, _, _ = \
         cmdgen.CommandGenerator().setCmd(
             cmdgen.CommunityData(self.snmp_rwcommunity),
             cmdgen.UdpTransportTarget((self.controller, 161)),
             (port_oid, rfc1902.Integer(self.CONTROL_OFF)),
         )
         if errorIndication or errorStatus != 0:
-            logging.debug("Failed to turn on PSU %s, exception: %s" % (str(psu_id), str(errorStatus)))
+            logging.debug("Failed to turn on PDU %s, exception: %s" % (str(outlet), str(errorStatus)))
             return False
         return True
 
-    def get_psu_status(self, psu_id=None):
+    def get_outlet_status(self, outlet=None):
         """
         @summary: Use SNMP to get status of PDU ports supplying power to PSUs of DUT
 
         DUT hostname must be configured in PDU port name/description. But it is hard to specify which PDU port is
-        connected to the first PSU of DUT and which port is connected to the second PSU.
+        connected to the first PDU of DUT and which port is connected to the second PDU.
 
         Because of this, currently we just find out which PDU ports are connected to PSUs of which DUT. We cannot
         find out the exact mapping between PDU ports and PSUs of DUT.
 
-        To overcome this limitation, the trick is to convert the specified psu_id to integer, then calculate the mode
+        To overcome this limitation, the trick is to convert the specified outlet to integer, then calculate the mode
         upon the number of PSUs on DUT. The calculated mode is used as an index to get PDU ports ID stored in
-        self.pdu_ports. But still, we cannot gurante that psu_id 0 is first PSU of DUT, and so on.
+        self.pdu_ports. But still, we cannot gurante that outlet 0 is first PDU of DUT, and so on.
 
-        @param psu_id: Optional. If specified, only return status of PDU port connected to specified PSU of DUT. If
+        @param outlet: Optional. If specified, only return status of PDU port connected to specified PDU of DUT. If
```

Comment:
s/PDU port/PDU outlet/
s/specified PDU of DUT/specified PSU of DUT/

Reply:
Thanks Joe, addressed.